assets/src/edit-story/components/library/panes/shared/libraryMoveable.js

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useCallback, useRef, useState } from 'react';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { editorToDataX, editorToDataY } from '../../../../units';
+import Moveable from '../../../moveable';
+import useSnapping from '../../../canvas/utils/useSnapping';
+import { useDropTargets } from '../../../dropTargets';
+import { useLayout } from '../../../../app/layout';
+import useInsertElement from '../../../canvas/useInsertElement';
+import { useCanvas } from '../../../canvas';
+import isMouseUpAClick from '../../../../utils/isMouseUpAClick';
+import InOverlay from '../../../overlay';
+import isTargetOutOfContainer from '../../../../utils/isTargetOutOfContainer';
+
+const TargetBox = styled.div`
+  position: absolute;
+  width: ${({ width }) => `${width}px`};
+  height: ${({ height }) => `${height}px`};
+`;
+
+function LibraryMoveable({
```

Comment:
Ah, I thought it felt weird but since previously the dragging worked like this, left it in. However, true that now we're using Moveable and can actually hide it. _Checking_